### PR TITLE
Comdex : add chain info, cmdx and cmst details

### PR DESCRIPTION
**Pull request by dheerajkd30**

### CODE REVIEW

File: chainConfig/comdex.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      ],
+      "jrpc": [
+        "https://rpc.comdex.one",
+        "https://comdex-rpc.polkachu.com",
+        "https://rpc-comdex.zenchainlabs.io",
+        "https://rpc-comdex-ia.cosmosia.notional.ventures",
+        "https://comdex-rpc.lavenderfive.com",
+        "https://rpc.comdex.chaintools.tech"
+      ],
+      "rpc": [""],
```
Comment (masterfung) [inline chainConfig/comdex.json:34]: Are there no rpc endpoints?

File: chainConfig/comdex.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      "currencies": [
+        {
+          "coinDenom": "CMDX",
+          "coinMinDenom": "ucmdx",
+          "coinDecimals": "6"
+        }
+      ],
+      "source": {
+        "sourceChannel": "channel-35",
+        "sourceIBCDenomToEvmos": "ibc/61DF64ADF65230540C14C63D64897BE08A3DC9A516A91425913F01240E2F432F",
```
[the remark this answers is not in the document]
Comment (dheerajkd30) [reply chainConfig/comdex.json:44]: Verifed this with IBC transfers and from tokens pairs here https://api.evmos.org/#/Query/TokenPairs, Looks good to go.

[the remark this answers is not in the document]
Comment (masterfung) [reply chainConfig/comdex.json:44]: Sounds good.

**Review by masterfung — APPROVED** [review]
Looks good.

**Review by masterfung — APPROVED** [review]

**Pull request merged**